public/js/popup.js

```diff
@@ -58,21 +83,30 @@ function warningModal(clone, options) {
         const errorCell = line.insertCell(2);
         errorCell.classList.add("highlight");
         errorCell.classList.add("msg");
+
         const positionCell = line.insertCell(3);
         positionCell.classList.add("position");
+        if (kind !== "short-identifiers" && kind !== "obfuscated-code") {
+            positionCell.addEventListener("mouseenter", (event) => fetchCodeLine(event.srcElement, unpkgFile, location, cache, lineId))
+        }
+
         if (value !== null) {
             errorCell.classList.add("clickable");
             errorCell.appendChild(document.createTextNode(value));
             errorCell.addEventListener("click", () => utils.copyToClipboard(value));
         }
 
+        const tooltip = utils.createDOMElement('div');
+        tooltip.classList.add('tooltip');
```

Comment:
```suggestion
        const tooltip = utils.createDOMElement('div', { classList: ['tooltip'] });
```

createDOMElement has a classList option.